### PR TITLE
Update fastify to v3

### PR DESCRIPTION
**Pull request by frikille**
Update dependencies
- [x] fastify to v3
- [x] fastify-plugin to v2
- [x] fastify-static to v2
- [x] fastify-websocket waiting to merge and release fastify/fastify-websocket#54

This PR also adds dependabot config and moves CI from Travis to Github Actions

### CODE REVIEW

**Comment by mcollina** [comment]
This has a conflict.

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: 1
+update_conigs:
+  package_manager: "javascript"
+  directory: "/"
+  update_schedule: "daily"
```
Comment (mcollina) [inline .dependabot/config.yml:5]: This is not correct.

**Comment by lazharichir** [comment]
Looking forward to this merge! Thanks for the PR! 👍

**Comment by admataz** [comment]
Note: CI is failing tests for node v10.x - seems to be related to the invalid default schema definition [here](https://github.com/mcollina/fastify-gql/blob/master/index.js#L109) that has an empty fields object.

**Pull request merged**